x/fswap/types/keys.go

```diff
@@ -17,6 +17,12 @@ const (
 	MemStoreKey = "mem_fswap"
 )
 
+var (
+	ParamsKey                 = byte(0x00)
+	SwappedKey                = byte(0x01)
+	SwappableNewCoinAmountKey = byte(0x02)
+)
+
 func KeyPrefix(p string) []byte {
 	return []byte(p)
 }
```

Comment:
1. I think, It'd better to move keys to `keeper/keys.go`
Because keeper is responsible of handling states with keys.

2. How about using `collections` util to generate key value in order to update to v0.50 style a little bit easier?
Here is example.
```go
import "cosmossdk.io/collections"

var (
	paramsKeyValue      = collections.NewPrefix(0)
	swappedKeyValue     = collections.NewPrefix(1)
)

func paramsKey() []byte {
	return paramsKeyValue.Bytes()
}
```

Reply:
About 2, thanks for your suggestion, I will try it.
And about 1, I find in cosmos0.50.0 version, all the keys.go are under types.

Reply:
About 1, all the keys are under type directory  in cosmos0.50.0 version.
About 2, I tried it before, but failed.

Reply:
About1, Okay, Thanks for letting me know.
About 2, I also failed to use whole collecions' feature, but was able to use key gen only.

Reply:
> About 2, I also failed to use whole collecions' feature, but was able to use key gen only.

But how could I use key to set/get, we cannot use `collections.NewItem`